docs/docs/api/Authentication/signup.md

```diff
@@ -16,72 +16,72 @@ color: '#E83E8C',
     "Email": string
 }
 ```
+
 ### Example Request
+
 #### Body
+
 ```json
 {
-    "Username": "Chris",
-    "Email": "hello@2ofClubs.app",
-    "Password": "password"
+  "name": "john",
```

Comment:
If I'm not mistaken, I believe we agreed that `name` wouldn't be here when signing up since our signup only takes in the `email` and `password`

Reply:
Fixed: Changes are reflected in recent commits